i915.c

```diff
@@ -199,6 +204,12 @@ static int i915_add_combinations(struct driver *drv)
 	metadata.priority = 3;
 	metadata.modifier = I915_FORMAT_MOD_Y_TILED;
 
+#ifdef USE_GRALLOC1
```

Comment:
this is not GRALLOC1 related, can remove this macro check

Reply:
modify accordingly.
please check.